### PR TITLE
Do the build of the wheel as not-root per our requirements

### PR DESCRIPTION
**Pull request by jcpunk**
This should fix up our container builds.  @vitodb

### CODE REVIEW

File: package/container/Dockerfile [hunk 1]
```diff
@@ -43,12 +43,12 @@ RUN python3 -m pip install --upgrade --prefix=/usr pip ; \
 
 # Install decisionengine requirements (only)
 #  do as one layer to simplify merges
-RUN git clone https://github.com/HEPCloud/decisionengine.git /tmp/decisionengine.git ; \
+RUN su - adm -s /bin/bash -c "git clone https://github.com/HEPCloud/decisionengine.git /tmp/decisionengine.git "; \
     cd /tmp/decisionengine.git ; \
-    python3 setup.py bdist_wheel ; \
+    su adm -s /bin/bash -c "python3 setup.py bdist_wheel" ; \
```
Comment (vitodb) [inline package/container/Dockerfile:48]: here it looks like we need
`su adm -s /bin/bash -c "cd /tmp/decisionengine.git; python3 setup.py bdist_wheel" ; \`
With this change I was able to build the docker container!

**Review by vitodb — CHANGES_REQUESTED** [review]

File: package/container/Dockerfile (hunk 1 above)
Comment (jcpunk) [reply package/container/Dockerfile:48]: `su` without the `-` should be leaving you in the directory you executed the command from.

Anyway, I can make that change easily enough.

Comment (vitodb) [reply package/container/Dockerfile]: My bad, on my local Dockerfile I had `su - adm` so I had to enforce the cd to get it working.
You are right, I'm able to build the docker container with your original update.

**Review by vitodb — APPROVED** [review]
All looks good.
With this update I'm able to build the docker container.
Thanks Pat!

**Pull request merged**